game.py

```diff
@@ -265,6 +295,8 @@ def _new_game(self, restore=False):
     def _ask_the_question(self):
         ''' Each game has a challenge '''
         self._timeout_id = None
+        for i in range (3):
```

Comment:
There is a space between `range` and `(3)` which doesn't go well with flake8

```suggestion
        for i in range(3):
```